Makefile

```diff
@@ -61,8 +61,10 @@ UI_DEPENDENCIES := search,mgmt,userprefs
 # freebsd is not supported for pie builds if CGO is disabled
 # see supported platforms at https://cs.opensource.google/go/go/+/master:src/internal/platform/supported.go;l=222-231;drc=d7fcb5cf80953f1d63246f1ae9defa60c5ce2d76
 BUILDMODE_FLAGS := -buildmode=pie
+BASE_IMAGE=gcr.io/distroless/base-debian12:latest-$(ARCH)
```

Comment:
Don't we need the equivalent of this replace also in cases where the stacker build/publish GH action is used?
In this file: https://github.com/project-zot/zot/blob/main/.github/workflows/publish.yaml

Reply:
Maybe we should make this a target in the Makefile itself.

Reply:
Updated stackerfile with new stacker version works fine.